src/commands/synthetics/utils.ts

```diff
@@ -754,6 +754,18 @@ export const parseVariablesFromCli = (
   return Object.keys(variables).length > 0 ? variables : undefined
 }
 
+export const validateDatadogSite = (datadogSite: string) => {
```

Comment:
Could you add tests for this function?

Reply:
This function achieves basic validation over the general format of a datadog site, so I just added basic testing. I'm not sure we want to do proper validation to reject `dqtqdqghq.com` for instance.